lib/parseAbi.test.ts

```diff
@@ -105,4 +124,40 @@ describe(parseAbi.name, () => {
     `),
     ).toThrow()
   })
+
+  it('handle human-readable cases without keywords', () => {
+    const abi = `
+      'constructor(string symbol, string name)',
+      'AccountLocked(address owner, uint256 balance)',
+      'transferFrom(address indexed from, address to, uint256 amount)',
+      'getUser(uint256 id) view returns (tuple(string name, address addr) user)',
+    `
+
+    const iface = parseAbi(abi)! as Interface
+
+    expect(iface).toBeA(Interface)
```

Comment:
that's why these tests should use snapshots... you almost don't assert anything here.